### DISCUSSION TITLE
fetching car data........

### DISCUSSION THREAD

**Discussion by Majd-ui** [theOehrly/Fast-F1#133] [Q&A]
Hello. I'm trying to load the FP3 session, when I start running it everything seem normal no error but it get stuck when it start fetching the car data.
this is what's in the console : 

`core           INFO 	Loading data for Saudi Arabian Grand Prix - Practice 3 [v2.2.0-alpha1]`
`api            INFO 	Using cached data for driver_info`
`core          ERROR 	Failed to load data from Ergast API!`
`Traceback (most recent call last):`
  `File "C:\Users\1234567\anaconda3\lib\site-packages\fastf1\core.py", line 1399, in _drivers_results_from_ergast`
    `data = ergast.fetch_results(`
  `File "C:\Users\1234567\anaconda3\lib\site-packages\fastf1\ergast.py", line 25, in fetch_results`
    r`eturn _parse_ergast(fetch_day(year, gp, day))[0][sel]`
`IndexError: list index out of range`
`api            INFO 	Using cached data for timing_data`
`api            INFO 	Using cached data for timing_app_data`
`core           INFO 	Processing timing data...`
`api            INFO 	Using cached data for session_status_data`
`api            INFO 	Using cached data for track_status_data`
`core        WARNING 	Driver 16: Lap timing integrity check failed for 1 lap(s)`
`api            INFO 	No cached data found for car_data. Loading data...`
`api            INFO 	Fetching car data...`

Is there a problem from Ergast API or?

PS: I'm using the same code I used in FP2

**Comment by theOehrly** [comment]
This should only happen if the session just finished recently, because ergast has no data yet. The error is expected. If you update to the latest stable version, you won't see it logged any more, as it doesn't matter.
